server/routes/auth.route.ts

```diff
@@ -0,0 +1,22 @@
+const Router = require('express')
```

Comment:
see, you should have a `user.route.ts` file instead of `auth.route.ts`, because it's more about users

Reply:
fix